### DISCUSSION TITLE
Sirius diagram editor plugin error

### DISCUSSION THREAD

**Discussion by fatimahghazali** [eclipse-sirius/sirius-desktop#438] [Q&A]
I followed the tutorial Basic Family,  I got the error as shown in the below picture. How to fix the sirius diagram editor plugin error?
![image](https://github.com/user-attachments/assets/bea52800-0949-413c-9ad6-6c7ba6901677)

**Comment by GlennPlou** [comment]
Hi,
It seems like you used a `description` feature somewhere. However, I don't see any `description` feature in the `basicfamily.ecore` metamodel. 
![image](https://github.com/user-attachments/assets/1eaf54e9-c268-4807-b18d-dc18e44215a8)
If you have made a mistake somewhere, you can still refer to the tutorial solution with `File > New > Example...` then in `Sirius Examples`: 
![image](https://github.com/user-attachments/assets/7cbba193-ed1d-44d7-8e8f-83cdd8af3653)

Best regards,
Glenn
